pkg/exporter/api/config.go

```diff
@@ -0,0 +1,37 @@
+// +build minimal
+
+package api
+
+// We export below types in order for cli package to be able to read it from configuration file.
+type LogConfig struct {
+	Level  string
+	Output string
+}
+
+type MetricsConfig struct {
+	Path string
+}
+
+type ServerConfig struct {
+	Protocol string
+	Host     string
+	Port     string
+}
+
+type ExporterConfig struct {
+	Port    string
+	Log     *LogConfig
+	Metrics *MetricsConfig
+}
+
+type Config struct {
+	ZotServer   ServerConfig
```

Comment:
Don't use project names for vars ("Zot...") 
What if we change the name of the project!

Reply:
Other places too!